src/openrct2-ui/windows/Finances.cpp

```diff
@@ -346,7 +346,7 @@ class FinancesWindow final : public Window
                 OnDrawMarketing(dpi);
                 break;
             case WINDOW_FINANCES_PAGE_RESEARCH:
-                OnDrawResearch(dpi);
+                WindowResearchFundingDraw(this, dpi);
```

Comment:
Same applies here. This goes for all functions that take `this` as their first argument.